lib/app.coffee

```diff
@@ -14,6 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ###
 
+Raven = require('raven')
+Raven.disableConsoleAlerts()
+Raven.config(
+	'https://56d2a46124614b01b0f4086897e96110:6e175465accc41b595a96947155f61fb@sentry.io/149239'
```

Comment:
For now I would just create the normal `config.coffee` file on the top level and move it there. Later it can be transformed into more advanced config or replaced with some build-time logic